Dockerfile.pqserver

```diff
@@ -1,12 +1,12 @@
 FROM python:3.11
 RUN pip install flask gunicorn sqlalchemy psycopg2
-WORKDIR /consdb-pq
-COPY python/lsst/consdb/pqserver.py python/lsst/consdb/utils.py /consdb-pq/
+WORKDIR /
+COPY python/lsst/consdb/__init__.py python/lsst/consdb/pqserver.py python/lsst/consdb/utils.py /consdb-pq/
 # Environment variables that must be set:
 # DB_HOST DB_PASS DB_USER DB_NAME or POSTGRES_URL
 
 # Expose the port.
 EXPOSE 8080
 
-ENTRYPOINT [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "2", "pqserver:app" ]
+ENTRYPOINT [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "2", "consdb-pq.pqserver:app" ]
```

Comment:
I'm kind of surprised that this works with a hyphen in the name.  If it doesn't, changing to just `consdb` should be fine, too.